plugins/action/system/shutdown.py

```diff
@@ -0,0 +1,214 @@
+# Copyright: (c) 2016-2018, Matt Davis <mdavis@ansible.com>
+# Copyright: (c) 2018, Sam Doran <sdoran@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import random
+import time
+
+from datetime import datetime, timedelta
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/ansible-collections/community.general/pull/700/commits/21bf25b0264a24e0cf5e2dc03f4c68a5b13d9401.